src/views/mainLayout.js

```diff
@@ -14,6 +14,27 @@ const mainLayout = data => {
   const metaDescription = data.meta?.socialDescription ?? getMessage('meta-desc-2')
   const pageUrl = `${AppConstants.SERVER_URL}${data.pathname ?? '/'}`
 
+  const shouldShowRecruitmentBanner = () => {
+    // Only show if ENVs are set and user language is set to "en"
+    if (
+      AppConstants.RECRUITMENT_BANNER_TEXT &&
+      AppConstants.RECRUITMENT_BANNER_LINK &&
+      getLocale().includes('en')
+    ) {
+      return true
+    }
+    return false
```

Comment:
since you're just boolean-ing, you could probably just return the `includes()` call directly.

```js
return AppConstants.RECRUITMENT_BANNER_TEXT &&
      AppConstants.RECRUITMENT_BANNER_LINK &&
      getLocale().includes('en');
```

Reply:
Removed this logic in favor of just calling `showRecruitmentBanner()`and returning an empty string if not satisfied. Revised in ad11567d72b884d0a90d9cd5b9a365d29d1569cb